src/app/models.py

```diff
@@ -59,10 +59,19 @@ class Camera(SQLModel, table=True):
     created_at: datetime = Field(default_factory=datetime.utcnow, nullable=False)
 
 
+class Pose(SQLModel, table=True):
+    __tablename__ = "poses"
+    id: int = Field(default=None, primary_key=True)
+    camera_id: int = Field(..., foreign_key="cameras.id", nullable=False)
+    azimuth: float = Field(..., ge=0, lt=360)
+    patrol_id: str | None = Field(default=None, max_length=100)
+
+
 class Detection(SQLModel, table=True):
     __tablename__ = "detections"
     id: int = Field(None, primary_key=True)
     camera_id: int = Field(..., foreign_key="cameras.id", nullable=False)
+    pose_id: int = Field(..., foreign_key="poses.id", nullable=True)
```

Comment:
maybe safer to allow None like for sequences 


```
    pose_id: int | None = Field(default=None, foreign_key="poses.id", nullable=True)
    sequence_id: int | None = Field(default=None, foreign_key="sequences.id", nullable=True)
```

Reply:
not sure to understand, here is cna't be bull

 pose_id: int = Field(..., foreign_key="poses.id", nullable=True)
 
 I just wanted to mimik the way it's done for sequence_id